### DISCUSSION TITLE
Student Pack Approved, But Still No Benefits After 3 Days – What's Going On?

### DISCUSSION THREAD

**Discussion by lsscrnls** [community/community#141513] [GitHub Education]
### Hi everyone,

Hey GitHub Team,

I applied for the GitHub Student Developer Pack three days ago and got the approval shortly after. However, I’m still waiting for access to the benefits. Three days seems a bit excessive for something that’s already been approved.

Can someone explain why it’s taking this long? I expect this to be resolved promptly as I need these tools for my ongoing projects.

Looking forward to a quick fix.

Thanks,
Lasse

**Comment by Sakib99-cyber** [comment]
Same Problem for me. Still waiting after 4 days. Need a quick solution. ( Approved but No Benefits After 4 Days)

**Comment by davevad93** [comment]
Hi @lsscrnls ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @lsscrnls was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
